.github/scripts/test-info.mjs

```diff
@@ -67,11 +70,13 @@ const components = {
             'AzureIotHubName',
             'AzureIotHubBindingsConsumerGroup',
         ],
+        sourcePkg: 'azure/eventhubs',
     },
     'bindings.azure.servicebusqueues': {
         conformance: true,
         certification: true,
         requiredSecrets: ['AzureServiceBusConnectionString'],
+        sourcePkg: 'azure/servicebusqueues',
```

Comment:
A lot of components, like servicebus, redis, etc, have a lot of code in a shared package. It would not be a fair calculation of code coverage if you didn't include those.

Please, allow passing multiple `sourcePkg` (type would be`string | string[]` to allow for an array too) and consider those too.

Reply:
The shared packages in `internal` folder are already taken into account. The `sourcePkg` is just the relative path of the component from the parent folder. 
See `-coverpkg=github.com/dapr/components-contrib/internal/component/${{ matrix.source-pkg }}/...,github.com/dapr/components-contrib/${SRC_PKG}/...`

Reply:
Done